feat(sourcemaps): Add more sourcemaps docs (#6667)
getsentry/sentry-docs: src/platform-includes/sourcemaps/upload/cli/javascript.mdx

</Note>

> If you decide to upload as an artifact bundle you should use Debug IDs, which are described more in detail in [artifact bundles vs. release bundles](ADD LINK).

Reviewer: ```suggestion
If you decide to upload source maps as an artifact bundle, you should use Debug IDs, which are described in detail in [artifact bundles vs. release bundles](ADD LINK).
```

#### Associating `release` with Artifact Bundle (Optional)

> If you are having troubles with Debug IDs, we suggest that you associate a release to an artifact bundle. _The release association will help Sentry figure out which artifact bundle(s) to use for unminification when Debug IDs are not present._

Reviewer: ```suggestion
If you're having trouble with Debug IDs, we suggest that you associate a release with an artifact bundle. _The release association will help Sentry figure out which artifact bundle(s) to use for unminification when Debug IDs are not present._
```

<Note>

> We strongly suggest to use the new Debug IDs system for uploading your artifacts since it provides more reliable unminification and the old release-based system will be faded out in the future.

Reviewer: ```suggestion
We strongly suggest that you use the new Debug ID system for uploading your artifacts since it provides more reliable unminification and the old release-based system will be faded out in the future.
```
